content/docs/command-reference/exp/init.md

```diff
@@ -19,10 +19,11 @@ usage: dvc exp init [-h] [-q | -v] [--run] [--interactive] [-f]
 ## Description
 
 `dvc exp init` helps you get started with DVC Experiments quickly. It reduces
-boilerplate DVC procedures by creating a `dvc.yaml` file that assumes standard
+repetitive DVC procedures by creating a `dvc.yaml` file that assumes standard
 locations of your input data, <abbr>parameters</abbr>, source code, models,
-<abbr>metrics</abbr> and [plots](/doc/command-reference/plots). These locations
-can be customized through the [options](#options) below or via
+<abbr>metrics</abbr> and [plots](/doc/command-reference/plots) (generating the
+boilerplate structure if needed). These locations can be customized through the
```

Comment:
> (generating the
boilerplate structure if needed)

This is only true of the inputs, which is one reason I suggested rewording the intro above to specify "inputs" and "outputs" (and why I think it's hard to separate #3440).

Reply:
> This is only true of the inputs

What about metrics.json, models/ , and plots/ @dberenbaum ? Sure, it's up to the user where to output any other artifact. Should we just note that?

Reply:
Oh I see. Those are only created with in `-i` mode... ⌛

Reply:
OK I changed the Description now. Actually there's even more nuance because with `-i` mode only the locations specified are created. Even input locations can be omitted and thus not generated. PTAL